uber/site_sections/registration.py

```diff
@@ -153,17 +155,15 @@ def delete(self, session, id, return_to = 'index?'):
                 session.delete_from_group(attendee, attendee.group)
                 message = 'Unassigned badge removed.'
             else:
-                #session.add(Attendee(**{attr: getattr(attendee, attr) for attr in [
-                #    'group', 'registered', 'badge_type', 'badge_num', 'paid', 'amount_paid', 'amount_extra'
-                #]}))
+                session.add(Attendee(**{attr: getattr(attendee, attr) for attr in [
+                    'group', 'registered', 'badge_type', 'badge_num', 'paid', 'amount_paid', 'amount_extra'
+                ]}))
                 session.assign_badges(attendee.group, attendee.group.badges + 1, attendee.badge_type)
-                Tracking.track(INVALIDATED, attendee)
-                #session.delete_from_group(attendee, attendee.group)
+                session.delete_from_group(attendee, attendee.group)
```

Comment:
BTW, I ripped out the "nothing ever gets deleted" functionality when doing this big refactor.  MAGFest also wants something similar, but I'm planning on going about it differently.  I'll be making a ticket for this soon, and I hope to have it implemented in time to launch MagClassic prereg.  Dunno if it'll be ready in time for Anthrocon; theoretically they could add it back in their own branch and/or override it in their plugin.